### PR TITLE
feat(lb): add lb frontend beta resource

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: scaleway/resource_lb_frontend_beta.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+				Required:     true,
+				ValidateFunc: validation.IntBetween(0, math.MaxUint16),
+				Description:  "TCP port to listen on the front side",
+			},
+			"timeout_client": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: diffSuppressFuncDuration,
+				ValidateFunc:     validateDuration(),
+				Description:      "Set the ",
```
Comment (kindermoumoute) [inline scaleway/resource_lb_frontend_beta.go]: doc

File: scaleway/resource_lb_frontend_beta.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validationUUIDorUUIDWithLocality(),
+				Description:  "The load-balancer backend id",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the backend",
```
Comment (kindermoumoute) [inline scaleway/resource_lb_frontend_beta.go]: frontend

File: scaleway/resource_lb_frontend_beta.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+	if err != nil {
+		return err
+	}
+
+	name, ok := d.GetOk("name")
+	if !ok {
+		name = getRandomName("lb-frt")
+	}
+
+	backendID := expandID(d.Get("backend_id"))
```
Comment (kindermoumoute) [inline scaleway/resource_lb_frontend_beta.go]: move to line 90

File: scaleway/resource_lb_frontend_beta.go [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+func resourceScalewayLbFrontendBetaUpdate(d *schema.ResourceData, m interface{}) error {
+	lbAPI, region, ID, err := getLbAPIWithRegionAndID(m, d.Id())
+	if err != nil {
+		return err
+	}
+
+	backendID := expandID(d.Get("backend_id"))
+	certificateID := scw.StringPtr(expandID(d.Get("certificate_id")))
```
Comment (kindermoumoute) [inline scaleway/resource_lb_frontend_beta.go]: func expandNullableString(s string) *string{
if s == "" {
		return nil
	}
return scw.StringPtr(s)
}

File: scaleway/resource_lb_frontend_beta.go [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+			State: schema.ImportStatePassthrough,
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"lb_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validationUUIDorUUIDWithLocality(),
+				Description:  "The load-balancer id",
```
Comment (QuentinBrosse) [inline scaleway/resource_lb_frontend_beta.go]: ID

File: scaleway/resource_lb_frontend_beta.go [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+			},
+			"backend_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validationUUIDorUUIDWithLocality(),
+				Description:  "The load-balancer backend id",
```
Comment (QuentinBrosse) [inline scaleway/resource_lb_frontend_beta.go]: ID

File: scaleway/resource_lb_frontend_beta.go [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+	lbAPI, region, ID, err := getLbAPIWithRegionAndID(m, d.Id())
+	if err != nil {
+		return err
+	}
+
+	err = lbAPI.DeleteFrontend(&lb.DeleteFrontendRequest{
+		Region:     region,
+		FrontendID: ID,
+	})
+
```
Comment (QuentinBrosse) [inline scaleway/resource_lb_frontend_beta.go]: newline

File: website/docs/r/lb_frontend_beta.html.markdown [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+-> **Note:** This terraform resource is flagged beta and might include breaking change in future releases.
+
+Creates and manages Scaleway Load-Balancer Frontends. For more information, see [the documentation](https://developers.scaleway.com/en/products/lb/api).
+
+## Examples
+    
+### Basic
+
+```hcl
+resource "scaleway_lb_frontend_beta" "backend01" {
```
Comment (QuentinBrosse) [inline website/docs/r/lb_frontend_beta.html.markdown]: frontend01

File: website/docs/r/lb_frontend_beta.html.markdown [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    name = "frontend01"
+    inbound_port = "80"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `lb_id`                       - (Required) The load-balancer ID this backend is attached to.
```
Comment (QuentinBrosse) [inline website/docs/r/lb_frontend_beta.html.markdown]: frontend

File: website/docs/r/lb_frontend_beta.html.markdown [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+- `backend_id`                  - (Required) The load-balancer backend ID this frontend is attached to.
+~> **Important:** Updates to `lb_id` or `backend_id` will recreate the backend.
```
Comment (QuentinBrosse) [inline website/docs/r/lb_frontend_beta.html.markdown]: frontend

File: website/docs/r/lb_frontend_beta.html.markdown [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+- `inbound_port`                - (Required) TCP port to listen on the front side.
+- `name`                        - (Optional) The name of the load-balancer frontend.
+- `timeout_client`              - (Optional) maximum inactivity time on the client side. (e.g.: `1s`)
```
Comment (QuentinBrosse) [inline website/docs/r/lb_frontend_beta.html.markdown]: Maximum

**Comment by jerome-quere** [comment]
Linked to #294

**Pull request merged**